src/js/common/polyfills.js

```diff
@@ -19,9 +19,16 @@ if (!Modernizr.classlist) {
 if (!Modernizr.dataset) {
   require('dataset');  // dataSet accessor support.
 }
-if (!Modernizr.fetch) {
```

Comment:
So...why take this check out?

Reply:
It's not needed, the fetch polyfill already has conditional logic for native fetch implementations: https://github.com/github/fetch/blob/master/fetch.js#L4